### PR TITLE
LT-1672: secure all with API key

### PR DESCRIPTION
**Pull request by GSailen**

### CODE REVIEW

File: src/Chest/appsettings.Deployment.json [hunk 1]
```diff
@@ -36,5 +36,9 @@
     "Properties": {
       "Application": "Chest"
     }
+  },
+  "ChestClientSettings": {
+    "ServiceUrl": "",
```
Comment (andrey-anokhin) [inline src/Chest/appsettings.Deployment.json]: Looks strange here + not used

File: src/Chest.Tests/Sdk/ChestFixture.cs [hunk 2]
```diff
@@ -10,97 +10,99 @@ namespace Chest.Tests.Sdk
...
-                        {
-                            throw;
-                        }
-                    }
-                }
-            }
-
-            return Process.GetProcessById(processId);
-        }
+//        private readonly string connectionString = "<pass_it_from_testsettings.json>";
```
Comment (andrey-anokhin) [inline src/Chest.Tests/Sdk/ChestFixture.cs]: Why it is commented?

File: src/Chest.Client/Compatibility/IMetadata.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+using System.Threading;
+using System.Threading.Tasks;
+using Chest.Client.AutorestClient.Models;
+using Microsoft.Rest;
+
+namespace Chest.Client.AutorestClient 
+{
+    public interface IMetadata
+    {
+        Chest.Client.IMetadata RefitClient { get; }
```
Comment (tarurar) [inline src/Chest.Client/Compatibility/IMetadata.cs]: Why do we use Refit dependency at all? We could have any name here, even something like `anylclient`.

Comment (aushakov) [reply src/Chest.Client/Compatibility/IMetadata.cs]: I did not change it to keep backward compatibility. This service client used in many services I don't want to change. I have refactored in this branch but could not apply the new client. https://github.com/lykkecloud/chest/commit/28761ecce0dd66704d34d3e5a43dd0465c38d225

**Pull request merged**